tai64/Cargo.toml

```diff
@@ -17,10 +17,12 @@ maintenance = { status = "passively-maintained" }
 [dependencies]
 chrono = { version = "0.4", optional = true, default-features = false }
 serde = { version = "1", optional = true, default-features = false }
+zeroize = { version = "1.1.0", optional = true, default-features = false, features = ["zeroize_derive"] }
 
 [features]
 default = ["std"]
 std = []
+zero_out = ["zeroize"]
```

Comment:
Suggest removing this: the feature can be called `zeroize`

Reply:
If I replace this with
```toml
[features]
zeroize = []
```
It results in an error `features and dependencies cannot have the same name: zeroize`

Reply:
Optional crates (somewhat confusingly) are already a cargo feature, so you can just delete this line and the `zeroize` "feature" should just work

Reply:
That works fine. Surely it's confusing